cli/pom.xml

```diff
@@ -40,6 +40,11 @@
       <artifactId>parsson</artifactId>
       <version>1.1.7</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
```

Comment:
why did you have to add this dependency additionally?
It is already a transitive dependency of all other jackson-* dependencies.
```suggestion
```

Reply:
I've removed it now.

Reply:
> did this also for the root pom.xml.

In dependencyManagement, I am fine with having additional dependencies managed for future usage.